rascal-lsp/src/main/rascal/lang/rascal/lsp/refactor/rename/Modules.rsc

```diff
@@ -115,15 +115,18 @@ bool isUnsupportedCursor(list[Tree] _:[*_, QualifiedName _, i:Import _, _, Heade
 }
 
 void renameDefinitionUnchecked(Define d:<_, currentName, _, moduleId(), _, _>, loc nameLoc, str newName, Renamer r) {
-    r.textEdit(replace(nameLoc, newName));
-
-    // Additionally, we rename the file
     if (currentName == newName) return;
     loc moduleFile = d.defined.top;
     pcfg = r.getConfig().getPathConfig(moduleFile);
-    loc relModulePath = relativize(pcfg.srcs, moduleFile);
-    loc srcFolder = [srcFolder | srcFolder <- pcfg.srcs, exists(srcFolder + relModulePath.path)][0];
-    r.documentEdit(renamed(moduleFile, srcFolder + makeFileName(forceUnescapeNames(newName))));
+    // Re-implement `relativize(loc, list[loc])`
```

Comment:
Is this an error for relativize? Shouldn't there be a fix for that function in the rascal project?

Reply:
That's would be a `fail` for `relativize`. I'm working on a change to the stdlib.

Reply:
This does a bit more since it also binds `srcDir`, that we need here.

Reply:
That fix lives here btw https://github.com/usethesource/rascal/pull/2252